### ISSUE TITLE
Joystick: handle shifts to a weird position on mousedown

### ISSUE THREAD

**Issue by alexcjohnson** [plotly/dash-daq#51]
On https://dash.plot.ly/dash-daq/joystick, the very first example does what you would expect - or at least, one of the two things you might expect: the handle moves so its center is exactly where the mouse pointer was when you moused down. (I might have wanted it to instead not move at all when you mouse down, just move when you drag, but I can see the argument for this behavior)

But the second and third examples don't behave correctly: mouse down and the handle moves to a point somewhere below where you clicked.
![joystick_bug](https://user-images.githubusercontent.com/2678795/59306410-28c60980-8c6a-11e9-8b81-4e81f8955b19.gif)

**Comment by yellenlab** [comment]
Hi, I'm also seeing the same issue when the layout changes, except it gets stuck at the top in my case.  Any progress on this issue?

**Comment by Karan-S-Mittal** [comment]
> On https://dash.plot.ly/dash-daq/joystick, the very first example does what you would expect - or at least, one of the two things you might expect: the handle moves so its center is exactly where the mouse pointer was when you moused down. (I might have wanted it to instead not move at all when you mouse down, just move when you drag, but I can see the argument for this behavior)
> 
> But the second and third examples don't behave correctly: mouse down and the handle moves to a point somewhere below where you clicked.
> ![joystick_bug](https://user-images.githubusercontent.com/2678795/59306410-28c60980-8c6a-11e9-8b81-4e81f8955b19.gif)

Hey @alexcjohnson, I wasn't able to replicate this bug. It's working fine after I tested it. on https://dash.plotly.com/dash-daq/joystick and in my local environment. adding gif for reference.
![issue51](https://user-images.githubusercontent.com/41485085/131254135-36058c17-7be7-41d8-a734-5fe4d9feafd6.gif)

**Comment by alexcjohnson** [comment]
Actually now I see the bug in all three examples, and in fact it's worse than it used to be, I can't even get the handle back to the middle no matter what I do. This happens in all browsers on my mac.

**Comment by Karan-S-Mittal** [comment]
Yes, It's odd behavior. I tried it again on my local dash application and it was showing the same behavior but in the web documentation, it is working fine.

**Comment by duemchen** [comment]
And now?
Today nov. 2024 in Firefox, Chrome and IPhone the same jump.
(But when I refresh the Side, its ok. Not the solution)
how to fix???

**Comment by duemchen** [comment]
Today 29.Oct. 2025 One year later
and I am the only man, who don't know, how to fix it

**Comment by Karan-S-Mittal** [comment]
I will take a look and share my findings.

**Comment by duemchen** [comment]
The same effects you can see on the original examples with label position and size
[Dash Python](https://dash.plotly.com/) > [Dash DAQ](https://dash.plotly.com/dash-daq) > Joystick
